pages/global-styles.mdx

```diff
@@ -0,0 +1,87 @@
+# Global styles
+
+Create a file called global.ts within the styles folder and paste the following code. Under GlobalStyle, general style rules are defined for the entire project.
+
+```
+import { createGlobalStyle } from 'styled-components';
+import { normalize } from 'styled-normalize';
+
+const GlobalStyle = createGlobalStyle`
+  ${normalize}
+
+  body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",
+    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+    "Helvetica Neue", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    scroll-behavior: smooth;
+  }
```

Comment:
suggestion: we could include also this text settings

```
	text-rendering: optimizeLegibility;
	font-feature-settings: "liga","kern"
	font-variant-numeric: proportional-nums;
```

Reply:
done!